app.py

```diff
@@ -136,7 +115,7 @@ def stop_server():
     return jsonify({"success": True, "message": "Server is shutting down..."})
 
 if __name__ == '__main__':
-
+    
```

Comment:
Please consider adding ` "files.trimTrailingWhitespace": true,` to your vscode `settings.json` this will prevent such things.